R/sparseR_prep.R

```diff
@@ -165,7 +165,7 @@ sparseR_prep <- function(formula, data, k = 1, poly = 1,
   rec_obj <- rec_obj_early
 
   # Create interactions if necessary
-  prec_obj <- prep(rec_obj, data = data)
```

Comment:
This is a typo, since the argument is named `training`, so this never did anything as `data` was swallowed silently